Makefile

```diff
@@ -61,6 +66,10 @@ clean:
 	@echo "Remove bin directory"
 	rm -rf ./bin
 
+clean-dirs:
+	rm -rf ${RUNDIRS}
+	pkill arp_proxy || true
```

Comment:
Shouldn't we kill arp_proxy before removing the log files?

Reply:
Yes, will move it up